lib/handler/status/durations.c

```diff
@@ -64,6 +69,11 @@ static void durations_status_per_thread(void *priv, h2o_context_t *ctx)
         ADD_DURATION(response_time);
         ADD_DURATION(total_time);
 #undef ADD_DURATION
+        if (agg_stats->stats.evloop_latency_ms.capacity <= ctx->thread_index) {
```

Comment:
I believe that this should be checking against `size`, `capacity` is internal to know whether re-allocation is necessary.